setup.py

```diff
@@ -74,7 +74,8 @@ def configuration(parent_package='', top_path=None):
 
 
 if __name__ == '__main__':
-    from numpy.distutils.core import setup, Extension
+    #from numpy.distutils.core import setup, Extension
+    from setuptools import setup, Extension
```

Comment:
Is setuptools.setup actually working here correctly? The setup call uses configuration argument that numpy.distutils.core.setup supports but configuration argument is non-standard feature, AWAIK.

Reply:
Resolved by PR #89